tests/test_cases/test_cocotb/test_handle.py

```diff
@@ -64,3 +64,26 @@ async def test_delayed_assignment_still_errors(dut):
         dut.stream_in_int <= "1010 not a real binary string"
     with assert_raises(TypeError):
         dut.stream_in_int <= []
+
+
+# identifiers starting with `_` are illegal in VHDL
+@cocotb.test(skip=cocotb.LANGUAGE in ["vhdl"])
+async def test_access_underscore_name(dut):
+    """Test accessing HDL name starting with an underscore"""
+    # direct access does not work because we consider such names cocotb-internal
+    with assert_raises(AttributeError):
+        dut._underscore_name <= 0
+    await cocotb.triggers.Timer(1, 'ns')
+    with assert_raises(AttributeError):
+        assert dut._underscore_name.value == 0
```

Comment:
These test the same thing. I think we should simplify this to simply:

```suggestion
    with assert_raises(AttributeError):
        dut._underscore_name
```

I don't think we can test `__setattr__`, because `dut._underscore_name = 0` would just silently set a private attribute.

Reply:
Done.